zerver/openapi/zulip.yaml

```diff
@@ -20363,6 +20363,14 @@ paths:
             type: boolean
             default: false
           example: true
+        - name: include_can_access_content
+          in: query
+          description: |
+            Include all the channels that the user has content access to.
+          schema:
+            type: boolean
+            default: false
+          example: true
```

Comment:
Whenever you change this file, you need to write a changes note.

`+            **Changes**: New in Zulip 10.0 (feature level 356).`